crypto_hal/monocypher/keys/key0.pub

```diff
@@ -0,0 +1,5 @@
+//Public key to verify signed binaries
```

Comment:
Does the key need to be passed to the build and this removed?

Reply:
Sure; maybe a macro pointing to the key file, and including in public_key.h? I'd still prefer having the keys in files, and not embedding the key itself into any macro/variable.